lib/model/QubitInformationObject.php

```diff
@@ -2959,18 +2962,21 @@ public function getInheritedReferenceCode()
 
     $identifier = implode(sfConfig::get('app_separator_character', '-'), $identifier);
 
-    if (isset($repository->identifier))
+    if ($includeRepoAndCountry)
```

Comment:
We could check if the repo exists in here, to avoid issues getting the identifier and to remove the repository check before getting the country code bellow.